tests/suites/0_stateless/20+_others/20_0001_planner_v2_cluster.result

```diff
@@ -0,0 +1,466 @@
+====SELECT_FROM_NUMBERS====
```

Comment:
Maybe we should take the difference test as other test case. This will help us known the detailed differences between standalone mode and cluster mode.

Reply:
Mostly about explain, have moved to a new test file.